data/campaigns/The_Rise_Of_Wesnoth/scenarios/17a_The_Dragon.cfg

```diff
@@ -237,7 +243,7 @@
             {TURNS_RUN_OUT}
 
             [gold_carryover]
-                carryover_percentage=40
+                carryover_percentage={ON_DIFFICULTY 60 50 40}
```

Comment:
This shouldn't vary based on difficulty, both here and later.

Reply:
aw man :-(